### ISSUE TITLE
No empty droplets detected

### ISSUE THREAD

**Issue by hviolaphd** [broadinstitute/CellBender#197]
Hi Stephen,

In this run there were no empty droplets detected. The CellRanger output suggests there should be. See below elbow plot from CellRanger. Attached PDF and log from the run. Any suggestions to fix? 

Thanks,
Hannah

![image](https://user-images.githubusercontent.com/112826043/232146774-ec8c8606-99d4-4954-bbac-4a584a8f03d7.png)

[6172.pdf](https://github.com/broadinstitute/CellBender/files/11236695/6172.pdf)
[6172.log](https://github.com/broadinstitute/CellBender/files/11236698/6172.log)

**Comment by sjfleming** [comment]
Yes!  My suggestion here is that `--total-droplets-included` is too small.  CellBender only comes up with a cell/no-cell call for those `--total-droplets-included`.  It looks like you used a value of 12000, and CellBender says all 12k are "non-empty", which I think is consistent with that UMI curve.  The droplets CellBender is really going to think are "truly empty" are those with UMI counts near 30, way down on the UMI curve.

I would suggest increasing `--total-droplets-included` to a number where you are absolutely sure that what lies past that on the UMI curve is really empty.  In this case, I can't see exactly without zooming in on that plot.  It looks like you might need to go out to 30k or even 40k.  Although CellRanger is saying the droplets between 10k and 20k are not cells, you'll find that CellBender will think some of them are cells.  And it's interesting to look at those droplets (after you do rigorous cell QC, because not everything CellBender says is "non-empty" is a high-quality cell).  Some of them might be a small cell type.

**Comment by sjfleming** [comment]
Can also now try to run v0.3.0, which might work better automatically without specifying anything for `--expected-cells` or `--total-droplets-included`

Potentially closed by #238
